kustomize/internalstorage/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
internalstorage doesn't need kustomize.yaml, we will only choose one of mysql or postgres in the installation.